pkg/controller/stacks/install/installjob.go

```diff
@@ -72,6 +73,12 @@ func createInstallJob(i v1alpha1.StackInstaller, executorInfo *stacks.ExecutorIn
 		namespace = o.Namespace
 	}
 
+	imagePullSecrets := i.GetSpec().ImagePullSecrets
+	imagePullPolicy := i.GetSpec().ImagePullPolicy
```

Comment:
For both of these I believe you will either need to check `GetSpec()` for `nil` or make the `spec` required on `StackInstall` and `ClusterStackInstall`. We have opted to make it required on a number of are other CRDs, and it seems like it may make sense for both of these as well.

Reply:
GetSpec has been replaced with getters and setters that into spec.